chapter_frame_test.go

```diff
@@ -29,129 +29,159 @@ func prepareTestFile() (*os.File, error) {
 }
 
 func TestAddChapterFrame(t *testing.T) {
-	tmpFile, err := prepareTestFile()
-	if err != nil {
-		t.Error(err)
-	}
-	defer os.Remove(tmpFile.Name())
-
-	tag, err := Open(tmpFile.Name(), Options{Parse: true})
-	if tag == nil || err != nil {
-		log.Fatal("Error while opening mp3 file: ", err)
-	}
-
-	chap := ChapterFrame{
-		ElementID:   "chap0",
-		StartTime:   0,
-		EndTime:     time.Duration(1000 * nanosInMillis),
-		StartOffset: 0,
-		EndOffset:   0,
-	}
-	tag.AddChapterFrame(chap)
-
-	if err := tag.Save(); err != nil {
-		t.Error(err)
-	}
-	tag.Close()
-
-	tag, err = Open(tmpFile.Name(), Options{Parse: true})
-	if tag == nil || err != nil {
-		log.Fatal("Error while opening mp3 file: ", err)
-	}
-	frame := tag.GetLastFrame("CHAP").(ChapterFrame)
-	if frame.ElementID != "chap0" {
-		t.Error(err)
-	}
-	if frame.StartTime != 0 {
-		t.Errorf("expected: %d, but got %s", 0, frame.StartTime)
-	}
-	if frame.EndTime.Seconds()*1000 != 1000 {
-		t.Errorf("expected: %d, but got %d", 1000, int(frame.EndTime.Seconds()*1000))
-	}
-}
-
-func TestAddChapterFrameWithTitle(t *testing.T) {
-	tmpFile, err := prepareTestFile()
-	if err != nil {
-		t.Error(err)
-	}
-	defer os.Remove(tmpFile.Name())
-
-	tag, err := Open(tmpFile.Name(), Options{Parse: true})
-	if tag == nil || err != nil {
-		log.Fatal("Error while opening mp3 file: ", err)
-	}
-
-	chap := ChapterFrame{
-		ElementID:   "chap0",
-		StartTime:   0,
-		EndTime:     0,
-		StartOffset: 0,
-		EndOffset:   0,
-		Title: &TextFrame{
-			Encoding: EncodingUTF8,
-			Text:     "chapter 0",
+	type fields struct {
+		ElementID   string
+		StartTime   time.Duration
+		EndTime     time.Duration
+		StartOffset uint32
+		EndOffset   uint32
+		Title       *TextFrame
+		Description *TextFrame
+	}
+	tests := []struct {
+		name            string
+		fields          fields
+		wantElementId   string
```

Comment:
Why not reusing `fields.ElementId`? In test cases the `fields.ElementId` and `wantElementId` are same